src/recoil/cartItems/selectors.test.ts

```diff
@@ -0,0 +1,156 @@
+import { renderHook } from '@testing-library/react';
+// import { act } from 'react';
+import { act } from 'react';
+import { RecoilRoot, useRecoilValue, useSetRecoilState } from 'recoil';
+
+import { cartItemsState } from './atoms';
+import {
+  deliveryPriceState,
+  orderTotalPriceState,
+  productTypesCountState,
+  purchaseTotalPriceState,
+  totalQuantityState,
+} from './selectors';
+
+import { TOTAL_PRICE_OVER_100000_DATA, TOTAL_PRICE_UNDER_100000_DATA } from '@/constants/mock';
+
+describe('selectors', () => {
+  describe('deliveryPriceState', () => {
+    it('총 결제금액이 100,000원 미만일 때, 배송비가 3000원이다.', () => {
+      const { result } = renderHook(
+        () => {
+          const deliveryPrice = useRecoilValue(deliveryPriceState);
+          const setCartItems = useSetRecoilState(cartItemsState);
+          return { deliveryPrice, setCartItems };
+        },
+        {
+          wrapper: RecoilRoot,
+        },
+      );
+
+      act(() => {
+        result.current.setCartItems(TOTAL_PRICE_UNDER_100000_DATA);
+      });
+
+      expect(result.current.deliveryPrice).toBe(3000);
+    });
+
+    it('총 결제금액이 100,000원 이상일 때, 배송비가 0원이다.', () => {
+      const { result } = renderHook(
+        () => {
+          const deliveryPrice = useRecoilValue(deliveryPriceState);
+          const setCartItems = useSetRecoilState(cartItemsState);
+          return { deliveryPrice, setCartItems };
+        },
+        {
+          wrapper: RecoilRoot,
+        },
+      );
+
+      act(() => {
+        result.current.setCartItems(TOTAL_PRICE_OVER_100000_DATA);
+      });
+
+      expect(result.current.deliveryPrice).toBe(0);
+    });
+  });
+
+  describe('orderTotalPriceState', () => {
+    it.each([
+      [TOTAL_PRICE_UNDER_100000_DATA, 21000],
+      [TOTAL_PRICE_OVER_100000_DATA, 100000],
+    ])('체크된 상품의 개수와 금액을 곱한 총 결제금액을 계산한다', (data, TOTAL_PRICE) => {
```

Comment:
명세의 역할을 할 수 있도록, 조금 더 원인과 결과를 명확히 적어주시면 조금 더 좋을 것 같아요!

```suggestion
  describe('orderTotalPriceState', () => {
    it.each([
      [TOTAL_PRICE_UNDER_100000_DATA, 21000],
      [TOTAL_PRICE_OVER_100000_DATA, 100000],
    ])('총 결제금액은 체크된 상품의 개수와 금액을 곱한 가격과 같아야 한다.', (data, TOTAL_PRICE) => {
```